db/migrate/20150418001125_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :password
+      t.string :email
+      t.string :first_name
+      t.string :phone_number
+      t.string :role
```

Comment:
Should we be adding all of these columns right off the bat? The `password` field may change if we use an authorization gem like Bcrypt, and the `role` field may also change depending on the type of authorization implemented. I'm always up for discussion as to why things are the way they are though :)